docs/README.md

```diff
@@ -60,58 +60,58 @@ The Media Communications Mesh (MCM) enables efficient, low-latency media transpo
 1. **Clone the repository**
 
    ```bash
-   $ git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
+   git clone https://github.com/OpenVisualCloud/Media-Communications-Mesh.git
    ```
 
 2. **Navigate to the Media-Communications-Mesh directory**
 
     ```bash
-    $ cd Media-Communications-Mesh
+    cd Media-Communications-Mesh
     ```
 
-3. **Install Dependencies**, choose between options `a)` or `b)`.
+3. **Install Dependencies**, choose between **OPTION A** or **OPTION B**.
 
-    a) Use all-in-one environment preparation script. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
+    - **OPTION A:** Use all-in-one environment preparation script. The script is designed for Debian and was tested under `Ubuntu 20.04` and `Ubuntu 22.04`, kernel version 5.15 environments.
 
     To use this option run the following command:
 
     ```bash
     sudo ./scripts/setup_build_env.sh
     ```
```

Comment:
in option B we recommend to reboot, we should do it in option A too. (the script asks about it, the readme should too)

Reply:
Added reboot information as requested.